bot/resources/tags/empty-json.md

```diff
@@ -0,0 +1,27 @@
+When creating a new JSON file you might run into the following error.
+
+`JSONDecodeError: Expecting value: line 1 column 1 (char 0)`
+
+In short this error means your JSON is invalid in it's current state.
```

Comment:
While true, this statement may be unnecessarily technical for some subset of the intended audience, who already know that their JSON isn't working. The line that follows it will probably be enough.

Reply:
Outdated. Has more info on the description below it now.